Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-SolarStorm_Activity_Behavior_Hunting_playbook.yml

```diff
@@ -0,0 +1,463 @@
+id: SolarStorm Activity Behavior Hunting playbook
```

Comment:
@kirbles19 @ShirleyDenkberg Does it make sense for the playbook name to contain the word playbook?